sites/all/modules/custom/training1_menu/templates/training1_menu_template.tpl.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * @file
+ * Template for list of links on pages theme/ and theme/list.
+ */
+?>
+
+<h2><?php print $title; ?></h2>
+<div>
+  <ol>
+    <?php foreach ($links as $name => $link): ?>
+        <?php print $item = '<li><a href=' . $link . '>' . $name . '</a></li>'; ?>
```

Comment:
Unnecessary '$item ='
In order to render a link use https://api.drupal.org/api/drupal/includes%21common.inc/function/l/7.x function